### PR TITLE
Uplift face of site

### LINKED ISSUE #21: Enhance the page on play.meshery.io
#### Current Behavior

Currently play.meshery.io doesn't have much description about the playground.
#### Desired Behavior

Enhance the page with more information about what playground is on playground.meshery.io 
#### Mockups

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR fixes #21 

**Screenshot for Reviewers**

Screen Shot 2022-11-15 at 10 52 29 PM

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: site/src/index.style.js [hunk 1]
```diff
@@ -7,10 +7,20 @@ export const GlobalStyle = createGlobalStyle`
...
 
+  @font-face {
+      font-family: "Qanelas Soft Regular";
+      src: url("https://db.onlinewebfonts.com/t/5173256ffa3e08c4cbed8986369258a6.eot"); /* IE9*/
+      src: url("https://db.onlinewebfonts.com/t/5173256ffa3e08c4cbed8986369258a6.eot?#iefix") format("embedded-opentype"), /* IE6-IE8 */
+      url("https://db.onlinewebfonts.com/t/5173256ffa3e08c4cbed8986369258a6.woff2") format("woff2"), /* chrome firefox */
+      url("https://db.onlinewebfonts.com/t/5173256ffa3e08c4cbed8986369258a6.woff") format("woff"), /* chrome firefox */
+      url("https://db.onlinewebfonts.com/t/5173256ffa3e08c4cbed8986369258a6.ttf") format("truetype"), /* chrome firefox opera Safari, Android, iOS 4.2+*/
+      url("https://db.onlinewebfonts.com/t/5173256ffa3e08c4cbed8986369258a6.svg#Qanelas Soft") format("svg"); /* iOS 4.1- */
+  }
```
[the remark this answers is not in the document]
Comment (Nikhil-Ladha) [reply site/src/index.style.js:18]: Here it is: https://github.com/meshery/play/tree/2434eba94f9623d408547935261ce3a6c9ceb4d2/site/src/assets/fonts/qanelas-soft

[the remark this answers is not in the document]
Comment (leecalcote) [reply site/src/index.style.js:18]: Oh, good. Also, I shouldn't have configured these assets to be hotlinked to a different site in the first place. :/

[the remark this answers is not in the document]
Comment (leecalcote) [reply site/src/index.style.js:18]: @warunicorn19, @Nikhil-Ladha if you get to this before I do, I won't be offended.

Comment (Nikhil-Ladha) [inline site/src/index.style.js:18]:
```suggestion
```

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you, thank you, thank you, @Nikhil-Ladha

**Pull request merged**